libraries/src/Filter/OutputFilter.php

```diff
@@ -47,7 +47,7 @@ public static function linkXHTMLSafe($input)
 	 */
 	public static function stringJSSafe($string)
 	{
-		$chars = preg_split('//u', $string, null, PREG_SPLIT_NO_EMPTY);
+		$chars = preg_split('//u', $string, 0, PREG_SPLIT_NO_EMPTY);
```

Comment:
```suggestion
		$chars = preg_split('//u', $string, -1, PREG_SPLIT_NO_EMPTY);
```
Use the same default value as defined in the documentation.